src/app.tsx

```diff
@@ -1,10 +1,14 @@
-import type React from 'react';
-import { st, classes } from './app.st.css';
+import InstagramPost from './components/InstagramPost';
+import { st, classes } from './style/main.st.css';
+import { POST_IMAGE, PROFILE_PICTURE, POST_DESCRIPTION, USER_NAME } from './constants';
```

Comment:
change the name of these to denote that they are sample images and texts

for example : SAMPLE_POST_IMAGE and SAMPLE_PROFILE_PICTURE